Makefile

```diff
@@ -16,17 +16,22 @@ help:
 	@echo ""
 	@echo "  Targets"
 	@echo ""
-	@echo "    deps       Install python packages"
-	@echo "    install    Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
-	@echo "    uninstall  Uninstall scripts and $(SHAREDIR)"
-	@echo "    docker     Build Docker image"
+	@echo "    deps-ubuntu	Install system packages (on Debian/Ubuntu)"
+	@echo "    deps       	Install python packages"
+	@echo "    install    	Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
+	@echo "    uninstall  	Uninstall scripts and $(SHAREDIR)"
+	@echo "    docker     	Build Docker image"
 	@echo ""
 	@echo "  Variables"
 	@echo ""
 	@echo "    PREFIX  Directory to install to ('$(PREFIX)')"
 
 # END-EVAL
 
+# Install system packages (on Debian/Ubuntu)
+deps-ubuntu:
+	apt-get install -y make python3 python3-pip python3-venv openjdk-8-jre-headless ghostscript
```

Comment:
How would I run this rule if `make` was not already installed?

Reply:
True, `make` is redundant at this point.

Reply:
Nor do we need python3-pip, as `pip` gets installed with a Python3 venv.